PatternPal/PatternPal.Extension/ViewModels/StepByStepListViewModel.cs

```diff
@@ -21,6 +20,7 @@
 
 namespace PatternPal.Extension.ViewModels
 {
+    // TODO comment
```

Comment:
@reviewer

Reply:
Didn't do since was a bit outside my knowledge, so documentation would be prefereable from @DanielvanDamme or @KnapSac